packages/block-library/src/embed/edit.js

```diff
@@ -203,8 +203,8 @@ const EmbedEdit = ( props ) => {
 	};
 
 	const onSubmitNative = ( value ) => {
-		// On native, the URL change is only notified when submitting
-		// so we have to explicitly set the URL.
+		// On native, the URL change is only notified when submitting,
+		// and not via 'onChange', so we have to explicitly set the URL.
```

Comment:
@hypest I made the comment update that was pending related to [this PR's comment](https://github.com/WordPress/gutenberg/pull/31343/files#r662069581).